mars/optimizes/utils.py

```diff
@@ -0,0 +1,110 @@
+from .ne import SUPPORT_OP, REDUCTION_OP, NeOptimizer
```

Comment:
I think the better way is to provide a base class for optimizer, implement some common methods like `compose`.